guide/07-working-with-bigdata/09-managing_big_data.ipynb

```diff
@@ -0,0 +1,1644 @@
+{
```

Comment:
<p>The <code><a href="https://developers.arcgis.com/rest/services-reference/enterprise/append-data.htm" target="_blank">append_data</a></code> operation appends tabular, point, line, or polygon data to an existing layer. The input layer must be a hosted feature layer. The tool will add the appended data as rows to the input layer. No new output layer will be created.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>